chpl/chpl-api/e2e/script/newman-run-dev.js

```diff
@@ -0,0 +1,76 @@
+const path= require('path'),
+async= require('async'),
+newman= require('newman');
+
+const collection_path= path.resolve('chpl/chpl-api/e2e/collections')
```

Comment:
These 2 const declarations are missing semicolons. Technically JS doesn't need them, but, it is best practice to avoid bugs and other statments in the file have them (as does other CHPL JS) so it is best for consistency as well.